avatar/avatar.css

```diff
@@ -0,0 +1,151 @@
+@import url('https://fonts.googleapis.com/css2?family=Fredoka&display=swap');
+
+:root{
+    --primary-color: #34D399;
+    --secondary-color: ;
+    
+}
+
+
+body{
+    margin: 0 auto;
+    font-family: 'Fredoka', sans-serif;
+}
+
+#tweet-img{
+    width:30px;
+}
+
+.line{
+    margin: 1rem;
+    padding: 0.1px;
+    background-color: gray;
+}
+
+iframe{
+    /* margin-left: 10%; */
+    margin-top: 1rem;
+    min-width: 90%;
+}
+
+.desc-txt{
+    
+    margin-top: 0.5rem;
+    font-weight: 600;
+}
+
+.container{
+    position: relative;
+}
+.sidenav-container{
+    position: fixed;
+}
+
+.main-container{
+    position: absolute;
+    left: 10rem;
+}
+
+.navigation{
+    background-color: var(--primary-color);
+    color: white;
+    padding: 0.1rem;
+    border-bottom-right-radius: 1rem;
+}
+
+.navigation .nav-brand{
+    font-weight: bold;
+    font-size: 2rem;
+    
+}
+
+.nav-pills{
+    text-align: right;
+}
+.list-non-bullet{
+    list-style: none;
```

Comment:
This you could have done in your reset.css file itself, you could have given `ul` `list-style: none` over there and then imported that file in all the files where you wanted to use that, you don't need to write an explicit class just for that.